mhkit/tests/utils/test_cache.py

```diff
@@ -93,7 +93,11 @@ def test_handle_caching_creates_cache(self):
         Asserts:
         - The cache file is successfully created at the expected file path.
         """
-        handle_caching(self.hash_params, self.cache_dir, data=self.data)
+        handle_caching(
+            self.hash_params,
+            self.cache_dir,
+            cache_content={"data": self.data, "metadata": None, "write_json": None},
+        )
```

Comment:
handle_caching reduced to only 5 inputs
https://pylint.readthedocs.io/en/stable/user_guide/configuration/all-options.html#max-args